terraform/modules/daily_snapshot/main.tf

```diff
@@ -65,7 +65,9 @@ locals {
     "echo 'export SLACK_NOTIF_CHANNEL=\"${var.slack_channel}\"' >> .forest_env",
     "echo 'export SNAPSHOT_BUCKET=\"${var.snapshot_bucket}\"' >> .forest_env",
     "echo 'export SNAPSHOT_ENDPOINT=\"${var.snapshot_endpoint}\"' >> .forest_env",
-    "echo 'export NR_LICENSE_KEY=\"${var.NR_LICENSE_KEY}\"' >> .forest_env",
+    "echo 'export NEW_RELIC_API_KEY=\"${var.NEW_RELIC_API_KEY}\"' >> .forest_env",
```

Comment:
Curious why don’t we have a template instead of those “echo” scripts.

Reply:
Yeah, so we can just interpolate the variables into it instead of echoing them one by one. 
I would consider doing as much as possible that was as opposed to embedding sh/bash 
into terraform configs.

Reply:
Let's add a follow-up issue.